packages/rich-text/src/to-tree.js

```diff
@@ -255,6 +255,7 @@ export function toTree( {
 					type: 'span',
 					attributes: {
 						'data-rich-text-placeholder': placeholder,
+						contenteditable: 'false',
```

Comment:
I don't know this part of the codebase particularly well, how does this change relate to the bug?

Reply:
Without this attribute, the placeholder would catch selection, with it, it cannot. I'll leave a comment.